texastribune/donations
# Multi entity edition

#### What's this PR do?
Adds functionality to make the app more dynamic and usable by different news orgs

#### Why are we doing this? How does it help us?
This is a first step in creating a one-for-all donation app for our different news entities.

#### How should this be manually tested?
For TT:
Visit the [staging site](https://donations-testing.herokuapp.com/donate)
Ensure the text looks as expected (nothing about Waco mentioned)
Attempt giving a donation (any amount/interval works)
Ensure our TT stripe account (in test mode) lists the new donation under "Transactions"
Ensure the donation was rippled to salesforce and has "texas" set as the newsroom

For WB:
Same as above except staging site is [here](https://waco-donations-2cb57b3c7271.herokuapp.com/donate), obviously the text should reflect Waco, look at the Waco Bridge account in stripe and the newsroom field in SF should be marked "waco"

#### How should this change be communicated to end users?
I'll keep the #local-waco channel up to date on progress

#### Are there any smells or added technical debt to note?
The /waco page needs to be removed and a handful of things can follow. There is more work to do to make this app fully dynamic but it would involve abstracting out some of the text bits in ways we haven't decided on yet.

#### What are the relevant tickets?
https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwS1XPty68eK4Ett/recsChD2Q2ybfU8eT?blocks=hide

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
